src/wechaty_plugin_contrib/daily_plugin.py

```diff
@@ -0,0 +1,30 @@
+"""daily plugin"""
+from datetime import datetime
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler     # type: ignore
+
+from wechaty import Wechaty
+from wechaty.plugin import WechatyPlugin
+
+
+class DailyPlugin(WechatyPlugin):
+    """
+    say something everyday, like `Daily Words`
+    """
+    @property
+    def name(self) -> str:
+        """get the name of the plugin"""
+        return 'dayily'
+
+    async def tick(self):
+        """time tick for the plugin scheduler"""
+        room = self.bot.Room.load('19961884194@chatroom')
```

Comment:
I believe the room id should be put into the `config` options of this plugin?

Reply:
Yes, I will make it been changed  later.